### PR TITLE
build osx-arm64 (Apple M1)

### PR DESCRIPTION
**Pull request by njzjz**
Closes #105.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by njzjz** [comment]
@conda-forge-admin, please rerender

**Comment by njzjz** [comment]
@conda-forge-admin rerender

**Comment by njzjz** [comment]
@conda-forge/lammps ready to review!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ source:
     sha256: 6d8bd8bc29cfebfddf215232acce910eaef403032ea8e276745455446875f1ac
     folder: lammps-plugins
   - fn: mlip.tar.gz
-    url: https://gitlab.com/ashapeev/interface-lammps-mlip-2/-/archive/master/interface-lammps-mlip-2-master.tar.gz
-    sha256: 37c5d9315cbc28e4b1cff36bbfac5b2b331fddcdf5fbcfd45e51c545137e969e
+    url: https://gitlab.com/ashapeev/interface-lammps-mlip-2/-/archive/dd71d892c590fe79c9800946dc8984de91c05480/interface-lammps-mlip-2-master.tar.gz
```
Comment (jan-janssen) [inline recipe/meta.yaml:25]: Can you explain the difference between the master branch and this specific commit?

Comment (jan-janssen) [reply recipe/meta.yaml:25]: Ok, I see it is just fixing the latest version of the `master` branch.

**Review by jan-janssen — APPROVED** [review]
Looks good to me

**Pull request merged**